java/arcs/sdk/android/storage/BUILD

```diff
@@ -15,7 +15,6 @@ arcs_kt_android_library(
         "//java/arcs/android/storage:proxy_message_android_proto",
         "//java/arcs/android/storage/database",
         "//java/arcs/android/storage/service",
-        "//java/arcs/android/storage/service:aidl",
```

Comment:
It looks like this import is required by blaze. Does it still work if you leave this in and just remove the idl_import_root statements?

Reply:
Yep, only leave one line change which removes idl_import_root = "." can still work.
So I bring back all "//java/arcs/android/storage/service:aidl" I removed before.

Reply:
Thanks for the fix!